docker/web/celery/scheduler/run.sh

```diff
@@ -10,8 +10,6 @@ else
 fi
 
 
-echo "==> $(date +%H:%M:%S) ==> Installing flower for monitoring <=="
-pip install 'celery[flower]'
-sleep 5
+sleep 10
```

Comment:
Why do we need this?

Reply:
Waiting for migrations to be run (this was already here, was removed this monday when doing testing with flower)